### PR TITLE
adding WIC.py script and IWIC.pdf

### PR DESCRIPTION
**Pull request by jrinks**
adding WIC.py script and IWIC.pdf

### CODE REVIEW

**Comment by michaelpkuhn** [comment]
I think the tests are failing because it is merging from another fork. @jordenk does that seem right?

**Comment by jordenk** [comment]
> I think the tests are failing because it is merging from another fork. @jordenk does that seem right?

I can't seem to reply to this as a thread. This is correct. The fork probably doesn't have the github environment variables set. @jrinks Can you make a branch from the main repo instead of forking the project? That will make CI easier. And it makes the commit history easier to read.

Branches in a repo are for adding features. Forking is used when the project will be maintained separately, or when long-term experimentation is desired.

File: WIC.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                "White", "Multi Racial", "Total_Participants", "Hispanic_Latino"]
+
+# create empty data frame with column names
+data = pd.DataFrame(columns=column_names)
+
+# read each page of the pdf pages 1 thru 97. Skip page 0 because it shows Statewide totals which we don't need
+pages = list(range(1, 97))
+
+# filepath to the PDF we want to read data from
+filepath = "C:/Users/User1/PycharmProjects/greater-chicago-food-despository/data_folder/IWIC.pdf"
```
Comment (brickman1444) [inline WIC.py:33]: Can we make this a relative path so that it will easily run on different computers without modifications?

File: WIC.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+# iterate thru pages 1 to 97
+for p in pages:
+    # read each page using pdfplumber
+    with pdfplumber.open(filepath) as pdf:
+        page = pdf.pages[p]
+        text = page.extract_text(x_tolerance=2, y_tolerance=0)
```
Comment (brickman1444) [inline WIC.py:40]: Can we add a comment explaining what x_tolerance and y_tolerance do? As a reader who is unfamiliar with pdfplumber I had to look up the documentation (https://pypi.org/project/pdfplumber/) to understand. I think something like "# extract_text() adds spaces where the horizontal distance between bits of text is greater than x_tolerance. It adds newline characters where the vertical distance between bits of text is greater than y_tolerance." would be helpful for future readers

File: WIC.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+# read each page of the pdf pages 1 thru 97. Skip page 0 because it shows Statewide totals which we don't need
+pages = list(range(1, 97))
+
+# filepath to the PDF we want to read data from
+filepath = "C:/Users/User1/PycharmProjects/greater-chicago-food-despository/data_folder/IWIC.pdf"
+
+# iterate thru pages 1 to 97
+for p in pages:
+    # read each page using pdfplumber
+    with pdfplumber.open(filepath) as pdf:
```
Comment (brickman1444) [inline WIC.py:38]: Does it take a more than a few seconds to run this script? It looks like we're reopening the pdf to just read one page. Opening files can be relatively slow, so if we need to speed up the script I think we could switch the lines "for p in pages:" and "with pdfplumber.open(filepath) as pdf:"

File: WIC.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+# We want to combine WIC1 and WIC2 columns into new column called WIC which will contain values such as "Total Women"
+# "Total Children" "Total INfants" and "LA Total"
+data.insert(2, "WIC", (data["WIC1"] + " " + data["WIC2"]))
+
+# delete WIC1 and WIC 2 columns
+data.drop(['WIC1', 'WIC2'], axis=1, inplace=True)
+
+# GCFD asked that we save the data to a csv and provide it to them
+# save as csv
+csv_filepath = "C:/Users/User1/PycharmProjects/greater-chicago-food-despository/data_folder/IWIC.csv"
```
Comment (brickman1444) [inline WIC.py:71]: Can we make this a relative path too so it's easier to run on different computers?

File: WIC.py [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+            # finds lines that start with LA
+            # appends to data
+            elif LA_Total_re.match(line):
+                new_line = (line.split(sep=" "))
+                new_line = County + new_line
+                data = data.append(pd.Series(new_line, index=data.columns), ignore_index=True)
+
+# Currently column WIC1 contains values "Total" and "LA" and WIC2 contains values such as "Women" "Children"
+# We want to combine WIC1 and WIC2 columns into new column called WIC which will contain values such as "Total Women"
+# "Total Children" "Total INfants" and "LA Total"
```
Comment (brickman1444) [inline WIC.py:63]: Is "INfants" a typo?

**Comment by michaelpkuhn** [comment]
Incorporated in #27

**Pull request closed without merging**